editor/src/components/editor/actions/actions.tsx

```diff
@@ -3094,64 +3098,125 @@ export const UPDATE_FNS = {
   },
   COPY_SELECTION_TO_CLIPBOARD: (
     action: CopySelectionToClipboard,
-    editorForAction: EditorModel,
+    editor: EditorModel,
     dispatch: EditorDispatch,
     builtInDependencies: BuiltInDependencies,
   ): EditorModel => {
-    return toastOnUncopyableElementsSelected(
-      'Cannot copy these elements.',
-      editorForAction,
-      false,
-      (editor) => {
-        // side effect 😟
-        const copyData = createClipboardDataFromSelection(editorForAction, builtInDependencies)
-        if (copyData != null) {
-          Clipboard.setClipboardData({
-            plainText: copyData.plaintext,
-            html: encodeUtopiaDataToHtml(copyData.data),
-          })
-        }
-        return {
-          ...editor,
-          pasteTargetsToIgnore: editor.selectedViews,
-          internalClipboard: {
-            styleClipboard: [],
-            elements: copyData?.data ?? [],
-          },
-        }
-      },
-      dispatch,
+    if (!isFeatureEnabled('Paste with props replaced')) {
+      return toastOnUncopyableElementsSelected(
+        'Cannot copy these elements.',
+        editor,
+        false,
+        (e) => {
+          // side effect 😟
+          const copyData = createClipboardDataFromSelection(e, builtInDependencies)
+          if (copyData != null) {
+            Clipboard.setClipboardData({
+              plainText: copyData.plaintext,
+              html: encodeUtopiaDataToHtml(copyData.data),
+            })
+          }
+          return {
+            ...e,
+            pasteTargetsToIgnore: e.selectedViews,
+            internalClipboard: {
+              styleClipboard: [],
+              elements: copyData?.data ?? [],
+            },
+          }
+        },
+        dispatch,
+      )
+    }
+
+    const canReparent = sequenceEither(
```

Comment:
You probably want `traverseEither` here, it's effectively the map and sequence combined.

Reply:
great idea! added in https://github.com/concrete-utopia/utopia/pull/3778/commits/7908d7a67e7de467aa4278d5f0c2aa940653db39